test-plans/template.md

```diff
@@ -0,0 +1,43 @@
+# Name
```

Comment:
Don't we use @LaviniaStiliadou's PDF template?

Reply:
no